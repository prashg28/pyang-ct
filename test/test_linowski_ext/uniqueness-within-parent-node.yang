 module unique {

   namespace "urn:org:example:xml:ct:unique";
   prefix "unique";

   import complex-types {prefix "ct"; }

   ct:complex-type  t1 {
   }

   ct:complex-type t1 {
   }

   ct:complex-type t2 {
   }

} // hw module
