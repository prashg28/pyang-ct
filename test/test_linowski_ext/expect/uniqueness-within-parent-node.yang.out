uniqueness-within-parent-node.yang:11: error: COMPLEX_TYPE_ALREADY_DEFINED
